packages/server-wallet/src/protocols/defunder.ts

```diff
@@ -79,8 +89,21 @@ export class Defunder {
       tx,
       channel.channelId
     );
+
     let didSubmitTransaction = false;
-    if (adjudicatorStatus.channelMode !== 'Finalized' && channel.hasConclusionProof) {
+    const shouldSubmitCollaborativeTx = Defunder._shouldSubmitCollaborativeTx(channel, objective);
+
+    /**
+     * The below if/else does not account for the following scenario:
```

Comment:
Am I right to interpret that this is a TODO? 

```suggestion
     * TODO: The below if/else does not account for the following scenario:
```

Reply:
It can be a TODO. Though a wallet that correctly treats the scenario described in the comment requires more changes than to just the code below the comment.